Base/res/html/tests/Attr-cloneNode.html

```diff
@@ -0,0 +1,34 @@
+<html>
+<body>
+  <pre id="out"></pre>
+  <script>
+    function log(s) {
+      document.getElementById("out").innerHTML += s + "\n"
+    }
+
+    window.onload = function() {
+      let e = document.createElement("div")
+      e.setAttribute("foo", "bar")
+      let attr = e.getAttributeNode("foo")
+      let clone = attr.cloneNode()
+
+      function dumpAttr(name, attr) {
+        log(name + ": " + attr)
+        log(name + ".ownerElement: " + attr.ownerElement)
+        log(name + ".namespaceURI: " + attr.namespaceURI)
+        log(name + ".localName: " + attr.localName)
+        log(name + ".name: " + attr.name)
+        log(name + ".value: " + attr.value)
+        log(name + ".specified: " + attr.specified)
+        log("")
+      }
+
+      dumpAttr("attr", attr)
+      dumpAttr("clone", clone)
+
+      log("attr === clone -> " + attr === clone)
+      log("attr.isEqualNode(clone) -> " + attr.isEqualNode(clone))
```

Comment:
As this PR introduces the `isEqualNode` functionality for `Attr`, would it be worth adding a sanity check for the opposite result?

```suggestion
      log("attr.isEqualNode(clone) -> " + attr.isEqualNode(clone))

      // this could go above
      e.setAttribute("boo", "baz")
      let otherAttr = e.getAttributeNode("boo")
      log("attr.isEqualNode(otherAttr) -> " + attr.isEqualNode(otherAttr))
```